### ISSUE TITLE
Version number for Frobby

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#1198]
`stdinc.h` is a file the Bjarke used in all his projects, but it shouldn't be included directly. Mathic, Mathicgb, and Memtailor also have their own. However, on the Debian version of Frobby this file doesn't exist.

https://github.com/Macaulay2/M2/blob/8a473f501318a6d505958ad6332e6618e5f16948/M2/Macaulay2/d/version.dd#L35

Added in https://github.com/Macaulay2/M2/commit/d6c19801fe8adb41a0950578571c20752f3364ea

**Comment by DanGrayson** [comment]
Maybe we just use these lines instead:
```
namespace constants {
  extern const char* const version;
}
```

**Comment by d-torrance** [comment]
I'm just hardcoding the frobby version number for now in the Debian package to get around this:

https://salsa.debian.org/science-team/macaulay2/-/blob/master/debian/patches/skip-frobby-version.patch

**Comment by DanGrayson** [comment]
Fixed in commit b0dc8d6a152c9d966ed515cb85acb03a6f464d5a (for now).

**Comment by DanGrayson** [comment]
By the way, further frobby development is up to us!  I have made a fork at https://github.com/Macaulay2/frobby , and I have various other bits of code to add to that for archival purposes.
